test/extended/dns/dns.go

```diff
@@ -114,6 +118,17 @@ func digForARecords(records map[string][]string, expect sets.String) string {
 	return probeCmd
 }
 
+func digForAAAARecords(records map[string][]string, expect sets.String) string {
+	var probeCmd string
+	fileNamePrefix := "test"
+	for name, ips := range records {
+		fileName := fmt.Sprintf("%s_endpoints_v6@%s", fileNamePrefix, name)
+		probeCmd += fmt.Sprintf(`[ "$$(dig +short +notcp +noall +answer +search %s AAAA | sort | xargs echo)" = "%s" ] && echo %q;`, name, strings.Join(ips, " "), fileName)
```

Comment:
Why do some of the other queries use +tcp in addition to +notcp, and save results to another file, but not A or AAAA queries?

Reply:
Hm, good question. I'm guessing the other query funcs test both TCP and UDP for extra code coverage. 

I think it's sufficient for this test to perform A and AAAA lookups via UDP only. How does that sound to you?

Reply:
I think it would be better for consistency sake to do it the same as the other queries.

Reply:
Maybe we could fix this up in a follow up? Before this PR, `digForARecords` only tested DNS via UDP, so it made sense to me to have `digForAAAARecords` do the same.